internal/common/common.go

```diff
@@ -42,6 +42,13 @@ const (
 	RoleStandby   Role = "standby"
 )
 
+// Roles enumerates all possible Role values
```

Comment:
This list needs maintaining to ensure we can set all possible label values to 0 on `stolon_keeper_{local,target}_role` before setting our active metric to be 1. This becomes clearer when you see the output of a single instances metrics:

![image](https://user-images.githubusercontent.com/3518874/56976029-f5239a00-6b69-11e9-8f9d-2d93d619c257.png)